### DISCUSSION TITLE
求宏碁暗影骑士擎2022：AN515-58-51R3

### DISCUSSION THREAD

**Discussion by shingo110** [daliansky/Hackintosh#412] [Q&A]
机型：AN515-58-51R3
处理器：第12代Intel® Core™ i5 12500H 处理器
显示屏：15.6“ FHD IPS 165Hz 100% sRGB，支持独显直连
显卡：NVIDIA® GeForce RTX™ 3060 6G
内存：16GB DDR4 3200MHz内存
硬盘：512GB PCIe4 NVMe SSD
网络连接：Killer Wi-Fi 6 AX1650i+Bluetooth 5.2+ Killer E2600 LAN

**Comment by merrystars** [comment]
您好，我已收到您的邮件。
